contracts/paymaster/src/forward_call.rs

```diff
@@ -23,28 +23,36 @@ pub trait ForwardCall {
             .with_callback(self.callbacks().transfer_callback(original_caller))
             .call_and_exit();
     }
-    
+
     #[callback]
     fn transfer_callback(
         &self,
         original_caller: ManagedAddress,
         #[call_result] result: ManagedAsyncCallResult<MultiValueEncoded<ManagedBuffer>>,
     ) -> MultiValueEncoded<ManagedBuffer> {
-        let initial_payments = self.call_value().all_esdt_transfers();
+        // TODO: use ManagedGetBackTransfers once rc1.6 is activated
+        let callback_payments = self.call_value().all_esdt_transfers();
```

Comment:
can you change this to egld_esdt_transfers ? as callBack might be called with eGLD as well.

Reply:
Fixed